### ISSUE TITLE
Make Error | Please help a newbie | msbuild

### ISSUE THREAD

**Issue by goddemnet** [dyc3/steamguard-cli#58]
My system: Arch
I followed: 
```
git clone --recursive https://github.com/dyc3/steamguard-cli.git
cd steamguard-cli
make
```

But at the end I get an error:
```
>>>> xbuild tool is deprecated and will be removed in future updates, use msbuild instead <<<<

MSBUILD: error MSBUILD0004: Too many project files specified

WARNING. Error reading msbuild project information, ensure that your input solution or project file is valid. NETCore and UAP projects will be skipped, only packages.config files will be restored.
Restoring NuGet package Newtonsoft.Json.12.0.3.
Adding package 'Newtonsoft.Json.12.0.3' to folder '/home/standart/steamguard-cli/SteamAuth/SteamAuth/packages'
Added package 'Newtonsoft.Json.12.0.3' to folder '/home/standart/steamguard-cli/SteamAuth/SteamAuth/packages'

NuGet Config files used:
    /home/standart/.config/NuGet/NuGet.Config

Feeds used:
    /home/standart/.nuget/packages/
    https://api.nuget.org/v3/index.json

Installed:
    1 package(s) to packages.config projects
echo "NEWTONSOFT_JSON_PATH="
NEWTONSOFT_JSON_PATH=
mcs -target:library -out:build/SteamAuth.dll -r: SteamAuth/SteamAuth/APIEndpoints.cs SteamAuth/SteamAuth/AuthenticatorLinker.cs SteamAuth/SteamAuth/Confirmation.cs SteamAuth/SteamAuth/SessionData.cs SteamAuth/SteamAuth/SteamGuardAccount.cs SteamAuth/SteamAuth/SteamWeb.cs SteamAuth/SteamAuth/TimeAligner.cs SteamAuth/SteamAuth/UserLogin.cs SteamAuth/SteamAuth/Util.cs SteamAuth/SteamAuth/Properties/AssemblyInfo.cs
error CS2005: Missing file specification for `/r:' option
make: *** [makefile:8: all] Error 1
```
I realized that the msbuild dependency is needed. 
Where do I get it, and how do I install it?
I searched all over the internet for it... And I'm already exhausted.

**Comment by goddemnet** [comment]
Maybe those of you who have gone through it. Will share a ready-made solution. 
Really hoping for help here!

**Comment by dyc3** [comment]
Make sure you have mono installed.

**Comment by dyc3** [comment]
@goddemnet try pulling the code and building it again. does it work?

**Comment by salahmak** [comment]
this was fixed recently on #59
